tests/test_gitignore_verification.py

```
@@ -0,0 +1,53 @@
+if not Path("codewiki").exists():
+    print("Please run this script from the project root directory.")
+    sys.exit(1)
+sys.path.insert(0, os.getcwd())
+
+from codewiki.src.be.dependency_analyzer.analysis.repo_analyzer import RepoAnalyzer
+
+temp_dir = Path(tempfile.mkdtemp(prefix="codewiki_verification_"))
+print(f"Creating test fixtures in: {temp_dir}")
+
+try:
+    subprocess.run(["git", "init", "-q"], cwd=temp_dir, check=True)
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)